astroid/interpreter/_import/spec.py

```diff
@@ -162,11 +161,6 @@ def contribute_to_path(self, spec, processed):
                 if os.path.isdir(os.path.join(p, *processed))
             ]
         # We already import distutils elsewhere in astroid,
```

Comment:
For example, to remove this comment 😅

Reply:
You can add `Co-authored-by: Marc Mueller <30130371+cdce8p@users.noreply.github.com>` when cleaning the squash message up :)